packages/config/src/Config.ts

```diff
@@ -152,6 +152,13 @@ export function getConfig(projectRoot: string, options: GetConfigOptions = {}):
       configWithDefaultValues.exp.currentFullName = getFullName(configWithDefaultValues.exp);
       // @ts-ignore: originalFullName not on type yet.
       configWithDefaultValues.exp.originalFullName = getFullName(configWithDefaultValues.exp);
+
+      if (configWithDefaultValues.exp.updates?.codeSigningCertificate) {
+        delete configWithDefaultValues.exp.updates.codeSigningCertificate;
+      }
+      if (configWithDefaultValues.exp.updates?.codeSigningMetadata) {
+        delete configWithDefaultValues.exp.updates.codeSigningMetadata;
+      }
```

Comment:
I see that checking before deleting is the convention in this file, but this is cleaner.
```suggestion
      delete configWithDefaultValues.exp.updates?.codeSigningCertificate;
      delete configWithDefaultValues.exp.updates?.codeSigningMetadata;
```